style.css

```diff
@@ -23,7 +31,20 @@ nav{
   text-align: center;
 }
 
-.mylogo a{
+.background--c {
+  position: absolute;
+  width: 100%;
+  height: 718px;
+  background-color: #fff;
+  background-image: url("Images/background.svg");
```

Comment:
-[ ] Nice Working on the previous reviewer's comments, like the previous review said your container does not have the round border, Kindly change the background image from `Images/background.svg` to `Images/background.png`, so the background becomes visible